sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/zuplo.yaml

```
@@ -0,0 +1,27 @@
+provider:
+  metadata:
+    name: Zuplo
+    category: Development
+    message: Make sure this Zuplo API key gets revoked, changed, and removed from the code.
+
+  rules:
+    - id: zuplo-api-key
+      rspecKey: S6753
+      metadata:
+        name: Zuplo API keys should not be disclosed
+      detection:
+        pre:
+          include:
+            content:
+              - zpka_
+        matching:
+          pattern: "\\b(zpka_[a-f\\d]{32}_[a-f\\d]{8})\\b"
+      examples:
+        - text: |
+            Authorization: Bearer zpka_513d294a9b5a44619dd6a02e55a20417_5e43e4d0
+          containsSecret: true
+          match: zpka_513d294a9b5a44619dd6a02e55a20417_5e43e4d0
```

I would suggest generating a completely different one. I use exrex for this, that you can find in the requirement/venv for the stats tools (https://github.com/SonarSource/languages-experimental-tooling/blob/gf/stats-tools/shannon/personal/gaetan-ferry/stats/requirements.txt)
```suggestion
        - text: |
            Authorization: Bearer zpka_db4ed7ea145619ba2c928f8988550d65_add4a1b8
          containsSecret: true
          match: zpka_db4ed7ea145619ba2c928f8988550d65_add4a1b8
```

---

I have generated a brand new token using `exrex.getone`.